src/bp-forums/groups.php

```diff
@@ -268,29 +268,7 @@ public function edit_screen( $group = false ) {
 			// Get the first forum ID
```

Comment:
@dipskakadiya I removed the code changes for not allowing forum to be associated to multiple groups as it's already being implemented in latest version of the plugin.